extensions/stats/plugin.cc

```diff
@@ -129,6 +157,9 @@ void PluginRootContext::report(bool is_tcp) {
                                             destination_node_info.namespace_());
   }
   istio_dimensions_.map(peer_node, request_info);
+  for (size_t i = 0; i < expressions_.size(); i++) {
```

Comment:
How does this handle changing a dimension.
I expect dimension_.map to take expression list as input so it does not end up redoing all the labels.

Reply:
It doesn't. We'd need to reorganize the implementation to avoid using macros and structs. This is just a minimal change given the constraints.